templates/repo/issue/view_content/comments.tmpl

```diff
@@ -378,12 +378,13 @@
 					<span class="badge{{if eq .Review.Type 1}} gt-bg-green gt-text-white{{else if eq .Review.Type 3}} gt-bg-red gt-text-white{{end}}">{{svg (printf "octicon-%s" .Review.Type.Icon)}}</span>
 					<span class="text grey muted-links">
 						{{if .OriginalAuthor}}
-							<span class="text black">
+							<span class="black">
```

Comment:
`black` class alone likely doesn't do anything and can be removed.

Reply:
It still needs `text black`, because the parent is `text grey`, an overriding game.